wdl-format/tests/format/ENCODE-DCC_chip-seq-pipeline/source.formatted.wdl

```diff
@@ -0,0 +1,3586 @@
+
+        Float spr_mem_factor = 20.0
+        Float spr_disk_factor = 30.0
+
+        Int jsd_cpu = 4
+        Float jsd_mem_factor = 0.1
+        Int jsd_time_hr = 6
+        Float jsd_disk_factor = 2.0
+
+        Int xcor_cpu = 2
+        Float xcor_mem_factor = 1.0
+        Int xcor_time_hr = 24
+        Float xcor_disk_factor = 4.5
+
+        Float subsample_ctl_mem_factor = 22.0
+        Float subsample_ctl_disk_factor = 15.0
+
+        Float macs2_signal_track_mem_factor = 12.0
+        Int macs2_signal_track_time_hr = 24
+        Float macs2_signal_track_disk_factor = 80.0
+
+        Int call_peak_cpu = 6
+        Float call_peak_spp_mem_factor = 5.0
+        Float call_peak_macs2_mem_factor = 5.0
+        Int call_peak_time_hr = 72
+        Float call_peak_spp_disk_factor = 5.0
+        Float call_peak_macs2_disk_factor = 30.0
+
+        String? align_trimmomatic_java_heap
+        String? filter_picard_java_heap
+        String? gc_bias_picard_java_heap
+    }
+
+    String pipeline_ver = "v2.2.2"
+    RuntimeEnvironment runtime_environment = {
+        "docker": docker,
+        "singularity": singularity,
+        "conda": conda,
+    }
+    RuntimeEnvironment runtime_environment_spp = {
+        "docker": docker,
+        "singularity": singularity,
+        "conda": conda_spp,
+    }
+    RuntimeEnvironment runtime_environment_macs2 = {
+        "docker": docker,
+        "singularity": singularity,
+        "conda": conda_macs2,
+    }
+
+    # read genome data and paths
+    if (defined(genome_tsv)) {
+        call read_genome_tsv { input:
+            genome_tsv = genome_tsv,
+            runtime_environment = runtime_environment,
+        }
+    }
+    File ref_fa_ = select_first([
+        ref_fa,
+        read_genome_tsv.ref_fa,
+    ])
+    File? bwa_idx_tar_ = if defined(bwa_idx_tar) then bwa_idx_tar else read_genome_tsv.bwa_idx_tar
+    File bowtie2_idx_tar_ = select_first([
+        bowtie2_idx_tar,
+        read_genome_tsv.bowtie2_idx_tar,
+    ])
+    File chrsz_ = select_first([
+        chrsz,
+        read_genome_tsv.chrsz,
+    ])
+    String gensz_ = select_first([
+        gensz,
+        read_genome_tsv.gensz,
+    ])
+    File? blacklist1_ = if defined(blacklist) then blacklist else read_genome_tsv.blacklist
+    File? blacklist2_ = if defined(blacklist2) then blacklist2 else read_genome_tsv.blacklist2
+    # merge multiple blacklists
+    # two blacklists can have different number of columns (3 vs 6)
+    # so we limit merged blacklist's columns to 3
+    Array[File] blacklists = select_all([
+        blacklist1_,
+        blacklist2_,
+    ])
+    if (length(blacklists) > 1) {
+        call pool_ta as pool_blacklist { input:
+            tas = blacklists,
+            col = 3,
+            runtime_environment = runtime_environment,
+        }
+    }
+    File? blacklist_ = if length(blacklists) > 1 then pool_blacklist.ta_pooled else if length(blacklists) > 0 then blacklists[0] else blacklist2_
```

Comment:
This may get handled by line length later, but should nested `if..then..else` constructs get line split regardless?

Reply:
The code you're looking at tries to put these constructs all on one line. They may end up getting split by other constructs, but that isn't accounted for in the `if` logic.

I do not have any concrete plans for how to change this. I definitely think it needs a fix, but I'm not sure what that is yet.